cmd/kubeadm/app/apis/bootstraptoken/v1/utils_test.go

```diff
@@ -27,6 +27,7 @@ import (
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	bootstrapapi "k8s.io/cluster-bootstrap/token/api"
+	"k8s.io/kubernetes/test/utils/ktesting"
```

Comment:
we shouldn't do this actually.
kubeadm is kind of standalone even if it still resides in k/k, so it should not import `k8s.io/kubernetes` at all if we want to extract it out of k/k eventually. i'm surprised import-boss is not [complaining about it](https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/.import-restrictions).

or maybe it is:
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/127882/pull-kubernetes-verify/1842712542653190144

but the error is not exactly clear.